### ISSUE TITLE
habr.ru post on Casper

### ISSUE THREAD

**Issue by Barkov-F** [rchain/bounties#669]
### habr.ru post on Casper

*Wrote a concise post on the RChain-flavoured Casper consensus protocol on habr.ru. Hope it will generate some attention to the project. 

*Estimated Budget of Task:* **1 RHOC per view in May**  \
*Estimated Timeline Required to Complete the Task:* **[Link to the post](https://habr.com/post/356428/)**  \
*Measure of Completion:* **Done**

[See CONTRIBUTING.md for details on budget and reward process.]

--  \
***Legal***  \
*Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws.*

**Comment by Ojimadu** [comment]
One RHOC per view? Do you think that is an appropriate metric for setting a budget?

**Comment by Barkov-F** [comment]
The post has got a thousand views in less than 24 hours and I hope @dckc can tell us if there was some effect on the docker container downloads or maybe PRs. What value do you find appropriate?

**Comment by Barkov-F** [comment]
I think if we establish per view payment for the content it will incentivize the content-makers to create high-value artifacts, ones that attract attention for long periods of time. Articles that you would like to add to your bookmarks

**Comment by Barkov-F** [comment]
Coop members can guide the marketing activities by playing with the REV per view rate.  How many REVs/view the Coop is ready to pay me for the publishing material on [this platform](https://habrastorage.org/storage/stuff/tm/mediakit_habrahabr_eng.pdf)? And how much value bring translations to the Nigerian pidgn published on a blog with a couple of subscribers?

**Comment by Barkov-F** [comment]
@patrick727 @pmoorman What do you think about "per view" compensation for posts, translations and such?

**Comment by jasoncruzzy** [comment]
@Barkov-F "per view" compensation might  not enhance performance of members at this early stage but it might be good if the "Rhoc per view" varies with different task. Great idea!

**Comment by Ojimadu** [comment]
"Per view" is a good performance-based option but I think there should be a cap on the number of views the coop should pay for at a given rate for X amount of time so as to prevent the use of traffic bots to a given content.

**Comment by Barkov-F** [comment]
[image: habraviewcount05102018.png]I think we should cap it at 10 thousand
views per day.  Of course, we have to mind what platform is being used.
Maybe more active ones should have a more generous limit.

**Comment by pmoorman** [comment]
@Barkov-F @Ojimadu feels like I need to add some color to my last statement:

> 
> Personally, I think this would be a great idea. Any form of performance-based compensation has my preference.
> 
> **Just write a suggestion and go for it.** It could act as an encouragement for other marketing work to be compensated similarly.

I see now that your suggested budget is 1 RHOC per view. 
Normally paid traffic is expressed in CPM (cost-per-mille == cost per 1,000 views). 

Here we're essentially suggesting a CPM of 1,000 RHOC or  **$1,640 CPM.**

By comparison, advertising on Facebook will cost in the range of anywhere from $5-50 CPM, and advertising on Google for targetted, high-value keywords would costs maybe double or triple that. 

...

**TL;DR:** I still think the _idea_ of CPM compensation is great, but I think the proposed _rate_ is ridiculous!

I would suggest adjusting it down with a factor 10x, or proving (or at least providing some evidence) that the costs are justified by Docker container downloads or Github PRs or some other measure of success.

**Comment by Barkov-F** [comment]
The extraordinary high CPM is a way how the Coop compensates the me (or any other content-creator) not only for the quality traffic acquisition but, primarily, for the creation of an artifact (a post, a video and so on) that later can cross-posted on other platforms and gather attention there. The effort to get the first view is much bigger that the latter efforts to get the first thousand  or the first million.

**Comment by Barkov-F** [comment]
And CPM of 1,000 RHOC is less than $1,500 now...

**Comment by pmoorman** [comment]
> The effort to get the first view is much bigger that the latter efforts to get the first thousand or the first million.

I would suggest you put in that extra little bit of effort to get those first 1 million views! In that case, we can work with a reasonable CPM indeed. 

Of course the exact idea of going for a **view-based compensation** is that it makes _views_ valuable, rather than _the artifact_ (duh!). If we value the artifact, we shouldn't have done CPM compensation to begin with!

**Comment by Barkov-F** [comment]
I'm paying with a [Ghost](https://rchain-world.herokuapp.com/untitled-c/), @ICA3DaR5 I can give you admin rights, if you want. I'm struggling with [DNS](https://rchain-world.herokuapp.com). The domain should be rchain.world

**Comment by Barkov-F** [comment]
@dckc @pmoorman @lapin7 @jimscarver  I'm glad we are having this conversation. I think in the future bounty system should adopt three payment models: _"per word"_, _"per view"_ and the _mixed_ one. 

**"per word"** model in my view is more suitable for the texts that will be more or less stable overtime like &#120588;-lang tutorial once &#120588;-lang turns 'Mercury', or architecture docs or other "RChain Coop Member's Handbook", speaking in thehum.com terms, thingies. 

**"per view"** is for marketing content like blog posts and so on. Setting different CPMs for different platforms (medium, habra.com, rchain.world, chinese and italian sites and so on) may help the Coop to guide the activities of RAMs. 

**mixed** is when the Coop approves of a **preset RHOCs/word** bounty for the creation a piece of content when there is a need _to produce that content quickly_, say, subtitles for the upcoming videos of @leithaus course on the computational calculi. But the author of the work still has the incentive to promote that content further because of the _per view_ part of the bounty contract.

**Comment by jasoncruzzy** [comment]
@Barkov F the three ways payment plan suggested is a welcome development.
It will ensure a better transparency system.

**Comment by Barkov-F** [comment]
@pmoorman @qelentium @lapin7 I solicit you to vote for budget and reward on the issue at the usual place

**Comment by Barkov-F** [comment]
One more vote is needed. Anyone? How much the Coop is ready to pay for the post? The audience is as close to the target as can get. Here is the media kit
[mediakit_habrahabr_eng.pdf](https://github.com/rchain/bounties/files/2079788/mediakit_habrahabr_eng.pdf)

![habr](https://user-images.githubusercontent.com/18509341/41090210-664c6a1a-6a4c-11e8-92d1-f10abbfe32c8.png)

**Comment by pmoorman** [comment]
@AyAyRon-P

**Comment by Barkov-F** [comment]
@kitblake Thanks for the reward vote. Can you put a budget vote in as well?
